_co-op-companies/B-google.md

```diff
@@ -9,6 +9,6 @@ complete: true
 members:
   - name: "Laura Stoyko"
     summary: "Laura will be working at Google this summer as an Engineering Practicum intern."
-    statement: "She will be working on the Tech Infrastrucutre Resource team, working on creating and populating databases in Java. She is very excited for this summer."
+    statement: "She worked on the Technical Infrastrucutre team, working on creating a user interface in Angular2/Typescript, while retrieving and processing the data for it in Java."

```

Comment:
typo: Infrastrucutre -> Infrastructure.
Typescript is generally spelled TypeScript as well.

Reply:
Probably Angular2 -> Angular 2 as well